adafruit_mcp2515/__init__.py

```diff
@@ -222,24 +222,24 @@ def __init__(
         auto_restart: bool = False,
         debug: bool = False
     ):
-        """[summary]
+        """A common shared-bus protocol.
+        :param ~busio.SPI spi: The SPI bus used to communicate with the MCP2515
+        :param ~digitalio.DigitalInOut cs_pin:  SPI bus enable pin
+        :param int baudrate: The bit rate of the bus in Hz, using a 16Mhz crystal. All devices on the bus must agree on this value. Defaults to 250000.
+        :param bool loopback: Receive only packets sent from this device, and send only to this device. Requires that `silent` is also set to `False`, but only prevents transmission to other devices. Otherwise the send/receive behavior is normal.
```

Comment:
please update (requires that `silent` be `True`)

Reply:
fixed